server/mscalendar/availability.go

```diff
@@ -235,17 +271,17 @@ func (m *mscalendar) setStatusFromCalendarView(user *store.User, status *model.S
 			m.Store.StoreUser(user)
 			err = m.Store.StoreUserActiveEvents(user.MattermostUserID, remoteHashes)
 			if err != nil {
-				return "", err
+				return "", errors.Wrap(err, fmt.Sprintf("Error in storing active events for user %s", user.MattermostUserID))
```

Comment:
You can use `fmt.Errof` and `%w` for error wrapping. I think it will be more convenient.